src/EditorFeatures/Core.Wpf/Adornments/AbstractAdornmentManager.cs

```diff
@@ -251,9 +251,9 @@ protected void UpdateSpans_CallOnlyOnUIThread(NormalizedSnapshotSpanCollection c
             AddAdornmentsToAdornmentLayer_CallOnlyOnUIThread(changedSpanCollection);
         }
 
-        protected bool ShouldDrawTag(SnapshotSpan snapshotSpan, IMappingTagSpan<GraphicsTag> mappingTagSpan)
+        protected bool ShouldDrawTag(SnapshotSpan snapshotSpan, IMappingTagSpan<GraphicsTag> mappingTagSpan, out SnapshotPoint? mappedPoint)
```

Comment:
```suggestion
        protected bool ShouldDrawTag(SnapshotSpan snapshotSpan, IMappingTagSpan<GraphicsTag> mappingTagSpan, out SnapshotPoint? mappedPoint)
```
```suggestion
        protected bool ShouldDrawTag(SnapshotSpan snapshotSpan, IMappingTagSpan<GraphicsTag> mappingTagSpan, [NotNullWhen(true)] out SnapshotPoint? mappedPoint)
```

(not sure if htat works for nullable value types).